docs/core/porting/project-structure.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Organizing Your Project to Support .NET Framework and .NET Core
+description: Organizing Your Project to Support .NET Framework and .NET Core
+keywords: .NET, .NET Core
+author: conniey
+manager: wpickett
+ms.date: 07/07/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 3af62252-1dfa-4336-8d2f-5cfdb57d7724
+---
+
+# Organizing Your Project to Support .NET Framework and .NET Core
+
+This article is to help project owners who want to compile their solution against .NET Framework and .NET Core side-by-side.  It provides several options to organize projects to help developers achieve this goal.  Here are some typical scenarios to consider when you are deciding how to setup your project layout with .NET Core.  They may not cover everything you want; prioritize based on your project's needs.
+
+* [**Combine existing projects and .NET Core projects into single projects**][option-xproj]
+  
+  *What this is good for:*
+  * Simplifying your build process by compiling a single project rather than compiling multiple projects, each targeting a different .NET Framework version or platform.
```

Comment:
Another goodness is that you don't have to managed source files for multiple projects because the project.json does the most part.  When there are multiple .csproj projects with shared source code, it is easy to miss some project(s) when adding/removing source files.